src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gapCalendar;
+  width: $widthCalendar;
+
+  &__body {
+    margin: 50px auto 0;
+  }
+
+  &__day {
+    position: relative;
+
+    width: $sizeDay;
+    height: $sizeDay;
+    padding: $paddingDay;
```

Comment:
no need to set padding here, because you positioning text inside with `position: absolute`